specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2023-04-01-preview/agfood.json

```diff
@@ -18095,6 +18044,27 @@
     }
   },
   "definitions": {
+    "ApiKeyAuthCredentials": {
+      "description": "ApiKeyAuthCredentials.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AuthCredentials"
+        },
+        {
+          "required": [
+            "apiKey"
+          ],
+          "type": "object",
+          "properties": {
+            "apiKey": {
```

Comment:
<img width="356" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/4725916/a82e59b9-5e77-4c39-85e7-42baf24a3480">

ApiKey model is being rendered in Swagger. may be it should be 

```
          "apiKey": {
            "description": "API key description",
            "schema": {
              "$ref": "#/definitions/KeyVaultProperties"
            }
          }